vip-integrations.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+use InvalidArgumentException;
+
+defined( 'ABSPATH' ) || die();
+
+require_once __DIR__ . '/vip-integration-helper/integrations/integration.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/integrations.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/block-data-api.php';
+
+class VipIntegrations {
+	public function setup() {
+		add_action( 'muplugins_loaded', [ $this, 'register' ], /* priority */ 4 );
+		add_action( 'muplugins_loaded', [ $this, 'activate' ], /* priority */ 6 );
+		add_action( 'muplugins_loaded', [ $this, 'integrate' ], /* priority */ 8 );
```

Comment:
What's the reason for staggering these across different priorities? If there's no good reason, I would just call one method and trigger the custom hooks there:

```
add_action( 'muplugins_loaded', [ $this, 'run' ] );

public function run() {
    do_action( 'vip_integrations_register' );
    do_action( 'vip_integrations_activate' );
...
}
```